packages/apache_tomcat/data_stream/access/_dev/test/system/test-default-config.yml

```
@@ -5,3 +5,6 @@ data_stream:
     paths:
       - "{{SERVICE_LOGS_DIR}}/localhost_access_log.*.txt"
     preserve_original_event: true
+numeric_keyword_fields:
+  - log.file.device_id
+  - log.file.inode
```

```suggestion
  - log.file.inode
  - log.file.idxhi
  - log.file.idxlo
  - log.file.vol
```

Those fields are also numbers.